config/default.json

```diff
@@ -49,6 +49,10 @@
     "apiKey": "some shared secret"
   },
   "timezone": "America/Los_Angeles",
+  "trendingScores": {
+    "activityWindow": 168,
+    "recalculationInterval": 60
```

Comment:
As noted in comments below, `activityWindow` uses hour units and `recalculationInterval` uses minutes. The idea was to make it easy to override in QA.